tailwind.config.ts

```
@@ -8,65 +8,101 @@ const config: Config = {
-        1000: '#121212'
+  plugins: [nextui({
+    themes: {
+      dark: {
+        colors: {
+          background: '#111315',
+          foreground: '#ffffff',
+          primary: '#23DCC8',
+          secondary: '#00CCFF',
+          success: '#23DCC8',
+          warning: '#F3BA2F',
+          danger: '#FF4954',
```

ESLint

[error] Unexpected trailing comma.

(@typescript-eslint/comma-dangle)